### PR TITLE
net-dns/bind-tools and net-dns/bind: 9.12.1 version bump

### PR DESCRIPTION
**Pull request by candrews**
Closes: https://bugs.gentoo.org/649294
Package-Manager: Portage-2.3.24, Repoman-2.3.6

### CODE REVIEW

File: net-dns/bind-tools/bind-tools-9.12.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+src_configure() {
+	local myconf=
+
+	if use urandom; then
+		myconf="${myconf} --with-randomdev=/dev/urandom"
+	else
+		myconf="${myconf} --with-randomdev=/dev/random"
+	fi
+
+	if use libedit ; then
```
[the remark this answers is not in the document]
Comment (candrews) [reply net-dns/bind-tools/bind-tools-9.12.1.ebuild]: It seems so - without this, if you specify USE="readline" and have libedit installed, you'll get an automagic dependency on libedit, which isn't good: https://bugs.gentoo.org/607400

File: net-dns/bind-tools/bind-tools-9.12.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+		--without-zlib \
+		--without-lmdb \
+		--disable-openssl-version-check \
+		$(use_enable ipv6) \
+		$(use_with idn) \
+		$(usex idn --with-idnlib=-lidnkit '') \
+		$(use_enable seccomp) \
+		$(use_with ssl openssl) \
+		$(use_with xml libxml2) \
+		$(use_with readline) \
```
[the remark this answers is not in the document]
Comment (candrews) [reply net-dns/bind-tools/bind-tools-9.12.1.ebuild]: fixed

File: net-dns/bind/bind-9.12.1.ebuild [hunk 3]
```diff
@@ -0,0 +1,423 @@
...
+
+	export BUILD_CC=$(tc-getBUILD_CC)
+	econf \
+		--sysconfdir=/etc/bind \
+		--localstatedir=/var \
+		--with-libtool \
+		--enable-full-report \
+		--without-readline \
+		$(use_enable caps linux-caps) \
+		$(use_enable filter-aaaa) \
```
[the remark this answers is not in the document]
Comment (candrews) [reply net-dns/bind/bind-9.12.1.ebuild]: I see, filter-aaa is always in bind, no longer configurable. Done.

[the remark this answers is not in the document]
Comment (candrews) [reply net-dns/bind/bind-9.12.1.ebuild]: Also added dnsrps

**Comment by idl0r** [comment]
Merged. Thanks!

**Pull request closed without merging**